app/kuma-dp/pkg/dataplane/envoy/remote_bootstrap_test.go

```diff
@@ -74,15 +87,26 @@ var _ = Describe("Remote Bootstrap", func() {
 							Name: "sample",
 						},
 					},
-					expectedBootstrapRequest: `
-                    {
-                      "mesh": "demo",
-                      "name": "sample",
-                      "adminPort": 4321,
-                      "dataplaneTokenPath": "/tmp/token",
-                      "dataplaneResource": "{\"type\":\"Dataplane\",\"mesh\":\"demo\",\"name\":\"sample\",\"creationTime\":\"0001-01-01T00:00:00Z\",\"modificationTime\":\"0001-01-01T00:00:00Z\"}"
-                    }
-`,
+					expectedBootstrapRequest: flatJson(`
```

Comment:
Why do we need `flatJson`? I was pretty sure `MatchJSON` handle new lines etc.

Reply:
You're right. It is redundant and I removed this function.